sklearn/utils/estimator_checks.py

```diff
@@ -71,6 +71,8 @@
                 'RANSACRegressor', 'RadiusNeighborsRegressor',
                 'RandomForestRegressor', 'Ridge', 'RidgeCV']
 
+# @glemaitre did the same thing, so basically things would be together
```

Comment:
Please don't add comments that are relevant to the PR but would not be relevant when merged. If you want to add comments about the PR, rather than about the code in its merged state, please do so in Github, not in the code.